pkg/apis/apps/validation/validation.go

```diff
@@ -88,6 +90,18 @@ func ValidatePersistentVolumeClaimRetentionPolicy(policy *apps.StatefulSetPersis
 	return allErrs
 }
 
+func ValidateVolumeClaimUpdatePolicy(policy apps.StatefulSetVolumeClaimUpdatePolicyType, fldPath *field.Path) field.ErrorList {
+	var allErrs field.ErrorList
+	switch policy {
+	case "":
```

Comment:
Does this policy field have a default value? If it is empty, is it necessary to check for errors?

Reply:
I'm following this guide https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api_changes.md#new-field-in-existing-api-version
> 5. In validation, validate the field if present

If the feature gate is not enabled, it is empty. Or it will have default value.